scriban.sln

```diff
@@ -3,32 +3,36 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
```

Comment:
Why the file was moved from src to top level? I would like to keep it where it was.

Reply:
Done.